data/mods/Xedra_Evolved/monsters/monsterattacks.json

```diff
@@ -279,6 +279,24 @@
     "effect": "summon",
     "effect_str": "mon_duplicator_pawn"
   },
+  {
+    "type": "monster_attack",
+    "attack_type": "melee",
+    "id": "migo_flesh_pull",
+    "cooldown": 7,
+    "move_cost": 106,
+    "accuracy": 6,
+    "blockable": false,
+    "damage_max_instance": [ { "damage_type": "bash", "amount": 5 }, { "damage_type": "cut", "amount": 7 } ],
+    "condition": { "not": { "u_has_effect": "maimed_arm" } },
+    "effects": [ { "id": "discombobulate", "duration": { "math": [ "_damage * 10" ] } } ],
```

Comment:
@GuardianDll can you look at this monster attack and see if I've done anything wrong?